packages/@aws-cdk/cloud-assembly-schema/lib/schema.ts

```diff
@@ -256,16 +258,12 @@ export interface MissingContext {
   /**
    * The provider from which we expect this context key to be obtained.
    */
-  readonly provider: string;
+  readonly provider: ContextProvider;
 
   /**
    * A set of provider-specific options.
    */
-  readonly props: {
-    account?: string;
-    region?: string;
-    [key: string]: any;
-  };
+  readonly props: ContextQueryProperties;
```

Comment:
This is actually a breaking change now since we don't allow `any`. 

I wonder how to approach this because as opposed to `metadata`, it doesn't make much sense to allow `any` here. Can we add a jsii exclusions for this specific violation? Would we need to remember to remove it after the release to enforce compatibility on this field going forward?

Reply:
Ah, good point. Yes, I think we should allow this technically-but-not-practically breaking change, and revert the exclusion later.

We should be resetting the exclusions file sometime soon anyway, it's excluding way too many things by now.